samples/bootloader/prj.conf

```diff
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
+CONFIG_NCS_SAMPLE_DEFAULTS=y
```

Comment:
Can any of the existing configs below be deleted?

Reply:
I could not identify any, and the ones for optimizing size are all still valid.

Reply:
Ok, great

Reply:
Perhaps add

    #CONFIG_NCS_SAMPLE_DEFAULTS=n

to the list for size optimizations.

Reply:
done